class/Unearthed Aiircana Red Mage.json

```diff
@@ -0,0 +1,1601 @@
+				"You can use this feature a number of times equal to your Intelligence modifier (a minimum of once). You regain all expended uses when you finish a long rest."
+			]
+		},
+		{
+			"name": "Grand Finale",
+			"source": "aiirRedMage",
+			"className": "Red Mage",
+			"classSource": "aiirRedMage",
+			"subclassShortName": "Sword Dancer",
+			"subclassSource": "aiirRedMage",
+			"level": 14,
+			"entries": [
+				"When you reach 14th level, you are able to perform an awe-inspiring dance with equally majestic swordplay, that is both beautiful and deadly.",
+				"As an action, you can expend three Balance dice, and for the duration of this turn, whilst moving within 5 feet of a creature, you can make a number of melee weapon attacks against that creature for every 10 feet you move. You deal an extra two Balance dice as force damage on your first hit. The number of attacks you can make with this feature is limited to a maximum of 5.",
+				"You can't use this feature again until you finish a long rest, unless you expend two Balance dice on top of the normal cost to use it again."
+			]
+		}
+	],
+	"spell": [
+		{
+			"name": "Jolt",
+			"level": 0,
+			"school": "V",
+			"time": [
+				{
+					"number": 1,
+					"unit": "action"
+				}
+			],
+			"range": {
+				"type": "point",
+				"distance": {
+					"type": "feet",
+					"amount": 30
+				}
+			},
+			"duration": [
+				{
+					"type": "instant"
+				}
+			],
+			"entries": [
+				"A shard of crystallized aether streaks toward a creature within range. Make a ranged spell attack against the target. On a hit, the target takes {@dice 1d8} force damage. This attack scores a critical hit if the number on the d20 is 19 or higher. A critical hit deals an extra {@dice 2d8} force damage to the normal hit, thus the critical hit deals a total of {@dice 3d8} force damage.",
+				"The spell's damage increases when you reach 5th level ({@dice 2d8} or {@dice 5d8} on a critical hit), 11th level ({@dice 3d8} or {@dice 7d8} on a critical hit), and 17th level ({@dice 4d8} or {@dice 9d8} on a critical hit)."
+			],
+			"scalingLevelDice": {
+				"label": "force damage",
+				"scaling": {
+					"1": "1d8",
+					"5": "2d8",
+					"11": "3d8",
+					"17": "4d8"
+				}
+			},
+			"damageInflict": [
+				"force"
+			],
+			"spellAttack": [
+				"R"
+			],
+			"source": "aiirRedMage",
+			"page": 9,
+			"components": {
+				"v": true,
+				"s": true,
+				"m": "a crystal medium"
+			},
+            "areaTags": [
+                "ST"
+            ]
+		},
+		{
+			"name": "Verflare",
+			"level": 4,
+			"school": "V",
+			"time": [
+				{
+					"number": 1,
+					"unit": "action"
+				}
+			],
+			"range": {
+				"type": "point",
+				"distance": {
+					"type": "feet",
+					"amount": 120
+				}
+			},
+			"duration": [
+				{
+					"type": "instant"
+				}
+			],
+			"classes": {
+				"fromClassList": []
+			},

```

Comment:
```suggestion
```

Reply:
The schema might not allow this to be empty now?